dtests/src/test/java/io/snappydata/hydra/connectionPool/tomcatConnPool.inc

```diff
@@ -0,0 +1,2 @@
+io.snappydata.hydra.connectionPool.SnappyConnectionPoolPrms-useTomcatConnPool = true;
```

Comment:
Use parameter with generic name and it can take names of connection pool to be used. e.g. useConnPool = ONEOF tomcat hikari FOENO